Gruntfile.js

```diff
@@ -104,7 +104,7 @@ module.exports = function (grunt) {
     let contentScopeInstall = 'echo "Skipping content-scope-scripts install";'
     let contentScopeBuild = 'echo "Skipping content-scope-scripts build";'
     // If we're watching the content scope files, regenerate them
-    if (grunt.option('watch')) {
+    if (Number(grunt.option('monitor'))) {
```

Comment:
Please could you add a note about this to the contribution guide?

Reply:
Added